src/app/lib/provider.ts

```diff
@@ -41,6 +44,7 @@ if (isProd) {
     global.ethProvider = MulticallWrapper.wrap(
       getDefaultProvider("mainnet", {
         alchemy: alchemyId,
+        exclusive: "alchemy",
```

Comment:
@andreitr also fyi, I added this to mute ethers Infura errors -- this should work fine so long as we don't have a fallback provider

Reply:
@stepandel sweet. I fixed this issue in the Scroll branch by going to tenant-specific Alchemy or JsonRPC providers but it will be good to have this in place in the meantime.